packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less

```diff
@@ -41,4 +37,40 @@
   &:first-of-type {
     margin-top: 0;
   }
+
+  > .vertical-tabs-pf {
+    > .vertical-tabs-pf-tab {
+      position: initial;
+      padding-left: 15px;
+    }
+  }
+}
+
+.vertical-tabs-pf.restrict-tabs {
+  .vertical-tabs-pf-tab {
+    display: none;
+
+    /* Show any active tab, tab that has an active descendant, or is force shown */
+    &.active,
+    &.active-descendant,
+    &.shown {
+      display: block;
+    }
+  }
+
+  /* Show siblings of the active tab */
+  &.active-tab {
+    > .vertical-tabs-pf-tab {
+      display: block;
+    }
+  }
+
+  /* Show the direct children of an active tab */
+  .vertical-tabs-pf-tab.active {
+    > .vertical-tabs-pf {
+      > .vertical-tabs-pf-tab {
```

Comment:
What if you take off this part of the selector, so this is 

```
.vertical-tabs-pf-tab.active {
  > .vertical-tabs-pf {
    display: block;
...
```

and....

Reply:
Since we are showing only the active tree, some of the list items are shown and some are not.  Its not all or nothing on the list.